lib/parsers/package-lock-parser.ts

```diff
@@ -296,12 +308,14 @@ export class PackageLockParser implements LockfileParser {
     while (depOrder.length) {
       const depKey = depOrder.shift() as string;
       const dep = depMap[depKey];
+      let treeSize = 1;
 
       // direction is from the dependency to the package requiring it, so we are
       // looking for predecessors
       for (const subDepPath of depGraph.predecessors(depKey)) {
         const subDep = depTrees[subDepPath];
         dep.dependencies[subDep.name] = subDep;
+        treeSize += depTreesSizes[subDepPath];
```

Comment:
I have no problem with this approach, but you can do it completely without `depTreesSizes`. I believe `dep.size += subDep.size + 1;` in every iteration is all you need, since you are builduing it gradually bottom-up. And every `subDep` is an instance of `PkgTree`, so it has `size available`.

Reply:
In this case each node will have size. But I want it to be only in the root node.